packages/@okta/i18n/src/properties/login.properties

```diff
@@ -955,3 +955,4 @@ oie.device.code.activated.title = Activate your device
 oie.device.code.activated.subtitle = Follow the instructions on your device to get an activation code.
 oie.device.code.activated.success.title = Device activated
 oie.device.code.activated.error.title = Device not activated
+new.key = value
```

Comment:
This is an example. Will remove before merge.

Reply:
Let's make sure we do this!